package.json

```diff
@@ -41,13 +41,15 @@
     "@nestjs/platform-express": "^10.0.0",
     "@nestjs/swagger": "^7.1.12",
     "@nestjs/throttler": "^5.1.2",
+    "@types/image-size": "^0.8.0",
```

Comment:
@CodeGiant1 Feedback:

`@types/image-size` Package has been deprecated and seems, you are trying to achieve the image sizing for the image upload which is also done with this package installed. `image-size`. we can just make use of this package 
`image-size` and remove the deprecated package so we don't create technical debt in this early stage.

Reply:
@CodeGiant1 this is still pending

Reply:
> @CodeGiant1 this is still pending

I am done with this task now and I have pushed my code